src/components/header/constants/locationData.js

```diff
@@ -0,0 +1,10 @@
+import dropDownIcon from '../../../assests/images/dropdown.svg'
+
+const locationData = {
+    name : "Sevashrama",
+    address : "Bangalore Railway Station",
+    img : dropDownIcon,
```

Comment:
Again one debatable question that is still in my mind should we take the icon as a prop in this scenario, may be or may not be but see your imgAlt is same as img(src), so you can use the same